test/integration/targets/acme_certificate_revoke/tasks/obtain-cert.yml

```diff
@@ -0,0 +1 @@
+../../acme_certificate/tasks/obtain-cert.yml
```

Comment:
Test targets should not depend on files from other targets, unless those targets are a dependency (ex: `setup_acme`). A change to `obtain-cert.yml` would only trigger the `acme_certificate` target to be tested.

The solution is to either have a copy of the file in each target that needs it, or put the original in the `setup_acme` target and symlink to it from the targets that need it. The test infrastructure knows about the dependency on the setup target, so changes to files there will trigger the necessary tests.

Sorry I didn't catch this in my earlier review.

Reply:
Ah, yes, a good point! I wanted to ask about this earlier, but then I forgot and when you didn't complain I somehow assumed it is probably ok ;-) Moving it to `setup_acme` is the best solution IMO.